src/components/Viewport.ts

```diff
@@ -1086,6 +1086,10 @@ export default class Viewport {
     this.setDefaultPanel();
     this.resize();
     this.moveToDefaultPanel();
+    if (this.options.renderOnlyVisible) {
```

Comment:
@WoodNeck 
this part is only for the first rendering.  I know that css is not injected even if resized.